eslint.config.js

```diff
@@ -0,0 +1,78 @@
+import eslint from '@eslint/js'
+import globals from 'globals'
+import playwright from 'eslint-plugin-playwright'
+import tseslint from 'typescript-eslint'
+
+const typescriptFiles = [
+    '**/*.ts',
+]
+
+export default [
+    // Generic EcmaScript.
```

Comment:
Move the different sets of generic rules to the end, so they override everything else and we know for certain they apply to all specified files.